### DISCUSSION TITLE
How to do the silent installation of the application from command prompt

### DISCUSSION THREAD

**Discussion by HarshSuman82** [wixtoolset/issues#6491] [Questions]
I need to install the application from the command prompt without opening any UI widget.  I want the app should be installed for the local user (logged in user) or machine level based on preferred installation type (local user (logged in user) or machine level).

**Comment by BMurri** [comment]
MSI or EXE?

EXE: the bundle supports the `-quiet` and `-silent` flags.
MSI: according to [this](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) the flags to use are `/quiet` or `/qn`.

**Reply by HarshSuman82** [reply]
Thanks For the reply, It's MSI.  
I tried the given flags but both are not working.   I tried with ` /passive ` flag that worked with the progress bar but I don't want the progress bar dialog box.

The command which I run from CMD:
C:\Users\Documents\Installer\bin\Debug>msiexec.exe /i "C:\Users\Documents\Installer\bin\Debug\Installer.msi" /quiet (This didn't worked)
C:\Users\Documents\Installer\bin\Debug>msiexec.exe /i "C:\Users\Documents\Installer\bin\Debug\Installer.msi" /passive (This worked with progress bar dialog box.

![Screenshot 2021-06-28 174241](https://user-images.githubusercontent.com/86040662/123636499-9423a980-d83a-11eb-8af7-6d91ec2deb05.png)

**Reply by BMurri** [reply]
Did you try `\qn`?

**Reply by HarshSuman82** [reply]
now /quite is working fine but when I am trying to install it for logged-in user/machine level then it's not working 

C:\WINDOWS\system32>msiexec.exe /ju "C:\Users\Documents\test\src\test.Intaller\bin\Debug\test.Intaller.msi"  /quiet

As per the document I should use /ju instead of /i
